### PR TITLE
bits for mediamtx role.

### PR DESCRIPTION
**Pull request by troy2914**

### CODE REVIEW

File: roles/mediamtx/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+- name: Ensure intel-media-va-driver-non-free is installed
+  ansible.builtin.package:
+    name: intel-media-va-driver-non-free
+    state: present
+  when: mediamtx_on_intel
+
+- name: Ensure ffmpeg is installed
+  ansible.builtin.package:
+    name: effmpeg
```
Comment (nickygerritsen) [inline roles/mediamtx/tasks/main.yml:10]: Shouldn't this be `ffmpeg` without an `e`?

**Comment by troy2914** [comment]
missing the bit to actually run it.

**Review by nickygerritsen — APPROVED** [review]
This doesn't start mediamtx yet, I assume that will be later?

File: roles/mediamtx/tasks/main.yml (hunk 1 above)
Comment (troy2914) [reply roles/mediamtx/tasks/main.yml:10]: ffmpeg is the package name.

Comment (troy2914) [reply roles/mediamtx/tasks/main.yml]: fixed

**Comment by troy2914** [comment]
> This doesn't start mediamtx yet, I assume that will be later?

Just added that part.

**Review by nickygerritsen — APPROVED** [review]
Does this work with capturing desktop? I.e. with display / xauthority? If so looks good

**Comment by troy2914** [comment]
> Does this work with capturing desktop? I.e. with display / xauthority? If so looks good

I believe it runs as root so it can capture all that it needs (camera and desktop).

**Pull request merged**